src/bridging/types.ts

```diff
@@ -345,8 +345,6 @@ export interface BridgeQuoteResults extends BridgeQuoteResult {
   bridgeCallDetails: BridgeCallDetails
 }
 
-export type GetErc20Decimals = (chainId: TargetChainId, tokenAddress: string) => Promise<number>
-
```

Comment:
Essentially you are changing the interface to require providers to provide full token details rather than just addresses for their supported tokens?

Reply:
Exactly: https://github.com/cowprotocol/cowswap/pull/5761